third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/ClassBuilder.java

```diff
@@ -203,6 +226,61 @@ private MetadataFile buildClassYmlFile(TypeElement classElement) {
     return classMetadataFile;
   }
 
+  // Does not set Inherited Methods or Inheritance as that information for Client classes is
+  // superfluous
+  // Sets updated Client summary with table of links
+  private void addClientClassInfo(
+      TypeElement classElement, MetadataFile classMetadataFile, RepoMetadata repoMetadata) {
+    MetadataFileItem classItem = new MetadataFileItem(LANGS, classLookup.extractUid(classElement));
+    classItem.setId(classLookup.extractId(classElement));
+    classItem.setParent(classLookup.extractParent(classElement));
+    addChildren(classElement, classItem.getChildren());
+    populateItemFieldsForClients(classItem, classLookup, classElement);
+    classItem.setPackageName(classLookup.extractPackageName(classElement));
+    classItem.setTypeParameters(classLookup.extractTypeParameters(classElement));
+    classItem.setInheritance(classLookup.extractSuperclass(classElement));
+    String summary = classLookup.extractSummary(classElement);
+    String summaryTable = createClientOverviewTable(classElement, repoMetadata);
+    classItem.setSummary(summaryTable + summary);
+    classItem.setStatus(classLookup.extractStatus(classElement));
+    classMetadataFile.getItems().add(classItem);
+  }
+
+  private String createClientOverviewTable(TypeElement classElement, RepoMetadata repoMetadata) {
+    String clientURI = classLookup.extractUid(classElement).replaceAll("\\.", "/");
+    String githubSourceLink =
+        repoMetadata.getGithubLink()
+            + "/"
+            + repoMetadata.getArtifactId()
+            + "/src/main/java/"
+            + clientURI
+            + ".java";
+    StringBuilder tableBuilder = new StringBuilder();
+    tableBuilder
+        .append("<table>")
+        .append("<tr>")
+        .append("<td><a href=\"")
+        .append(githubSourceLink)
+        .append("\">GitHub Repository</a></td>")
+        .append("<td><a href=\"")
+        .append(repoMetadata.getProductDocumentationUri())
+        .append("\">Product Reference</a></td>");
+    if (repoMetadata.getRestDocumentationUri().isPresent()) {
```

Comment:
Very-optional syntax sugar, just FYI:
You could choose to avoid the `get()` by using `ifPresent()`:
```java
repoMetadata.getRestDocumentationUri().ifPresent(data -> 
  tableBuilder.append("<td>...").append(data)...("</td>"));
```

Reply:
Oh cool! Thank you - updated (good to practice!)